Domain/Validators/TuitionPartnersFilterRequestValidator.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Domain.Validators;
+
+public abstract class TuitionPartnersFilterRequestValidator<T>
+{
+    protected TuitionPartnersFilterRequestValidator()
```

Comment:
Do we need this empty constructor?

Reply:
I suspect not - I think I just followed "TuitionPartnerSearchRequestValidator.cs", but think that the constructor can be removed from both.  I'll update that.

Reply:
Removed these empty constructors.